### ISSUE TITLE
Disqus comments not working

### ISSUE THREAD

**Issue by ketanbhatt** [abakan/ablog#33]
By default disqus had a problem connecting to their server. 

I saw that if I remove "disqus_title", " disqus_URL" and "disqus_identifier" from templates/page.html, it worked fine. 
Please look into it

**Comment by abakan** [comment]
Was it a single page or all pages? If a single page, can you copy contents? Was it when you served files locally?

**Comment by ketanbhatt** [comment]
I tried it on just one post because I was in the test period. 

Happened both locally and on github pages. 

When I removed the said lines from the HTML, it started working normally.

**Comment by abakan** [comment]
Maybe your title had a special character in it. Can you point to rst source if it's available online?

**Comment by ketanbhatt** [comment]
I thought too that was the case and so I removed everything I thought would be causing a problem. 
Didn't help. 

Only removing those three lines from the templste helped. 
When I used Jekyll to build my blog, I manually copied the for from disqus's website to my template. That code didn't have those lines too, and that's how I thought of removing them from ablog templates too. 

No, the source code isn't available online now, but I can upload if that will help, but I don't think there is a problem with that otherwise it wouldn't have worked at all. 

Maybe do some tests at your end to check this out?

**Comment by abakan** [comment]
Source .rst would help debugging. Was this the title "Git Your B'Day Caps Ready!"? \' needs escaping.

**Comment by ketanbhatt** [comment]
I removed the ' completely after reading the guidelines given in disqus community. 

And again, it shouldn't have worked for the same title when I removed the three lines of code. 

I think I will try it again with two three posts and let you know with screenshots

**Comment by ketanbhatt** [comment]
So, I tested ablog. I made no changes to any rst file. "First Post" is made out of the box with "ablog start". "New Post" is made with "ablog post <file-name>".

I tested both the pages, same trouble. Then I edited the templates for one of them, here are the results:

FIRST POST with no edits:
![first post testing ablog documentation](https://cloud.githubusercontent.com/assets/9729957/7117783/b393b3ec-e218-11e4-8e4f-b5169b9523e2.png)


NEW POST with disqus tags removed (except disqus shortname):
![new post testing ablog documentation](https://cloud.githubusercontent.com/assets/9729957/7117807/d5c1de30-e218-11e4-9018-56d91bef9428.png)
